include/picongpu/particles/atomicPhysics/rateCalculation/BoundFreeFieldTransitionRates.hpp

```diff
@@ -39,77 +40,108 @@ namespace picongpu::particles::atomicPhysics::rateCalculation
     template<atomicPhysics::enums::ADKLaserPolarization T_ADKLaserPolarization>
     struct BoundFreeFieldTransitionRates
     {
-        /** rate for bound-free field ionization transition of ion depending on the local electric field
-         *
-         * @tparam T_ChargeStateDataBox instantiated type of dataBox
-         * @tparam T_AtomicStateDataBox instantiated type of dataBox
-         * @tparam T_BoundFreeTransitionDataBox instantiated type of dataBox
+    private:
+        struct ResultStruct
+        {
+            // unitless
+            PMACC_ALIGN(effectivePrincipalQuantumNumber, float_X);
+            // in elementary charge
+            PMACC_ALIGN(screenedCharge, float_X);
+        };
+
+        /** get input required for calling rate formula
          *
-         * @param eField E-field vector, in internal units
-         * @param ionizationPotentialDepression, eV
+         * @param ionizationPotentialDepression, in eV
          * @param transitionCollectionIndex index of transition in boundBoundTransitionDataBox
          * @param atomicStateDataBox access to atomic state property data
          * @param boundBoundTransitionDataBox access to bound-bound transition data
-         *
-         * @return unit: 1/picongpu::sim.unit.time()
          */
-        template<
-            typename T_EFieldType,
-            typename T_ChargeStateDataBox,
-            typename T_AtomicStateDataBox,
-            typename T_BoundFreeTransitionDataBox>
-        HDINLINE static float_X rateADKFieldIonization(
-            // internal units
-            T_EFieldType const eFieldNorm,
-            // eV
+        template<typename T_ChargeStateDataBox, typename T_AtomicStateDataBox, typename T_BoundFreeTransitionDataBox>
+        HDINLINE static ResultStruct getInput(
```

Comment:
Make this the constructor of `RateFormulaVariables`.

Reply:
done